bundles/org.openhab.binding.groupepsa/README.md

```diff
@@ -0,0 +1,28 @@
+# Groupe PSA Binding
+
+Binding to retrieve information via the Groupe PSA Web API for cars from Opel, Peugeot, Citroen, DS and Vauxhall.
+
+## Supported Things
+
+Groupe PSA Web Api Bridge - The Thing to auto discover your cars
+Groupe PSA Car - The actual car thing.
+
+## Discovery
+
+Use the "Groupe PSA Web Api bridge" to auto discover your cars. You need to select the brand for the bridge binding and only cars for the brand will be auto discovered. If you need to add for multiple brands or multiple different users, add multiple bridges.
+
+## Binding Configuration
```

Comment:
If I see correctly, you binding doesn't have a configuration at binding level. You might mean the bridge configuration.